sale_order_lot_selection/wizards/stock_lot_add_to_sale_order.py

```
@@ -0,0 +1,161 @@
+        self.ensure_one()
+        if not self.partner_id:
+            raise ValidationError(_("Please select a Customer"))
+        self.sale_order_id = self.env["sale.order"].create(
+            {
+                "partner_id": self.partner_id.id,
+                "order_line": self._get_vals_for_add_lot_in_sale_order_line(),
+            }
+        )
+        return self._action_open_sale_order()
+
+    def _get_vals_for_add_lot_in_sale_order_line(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_get_vals_for_add_lot_in_sale_order_line`

(ANN202)